### PR TITLE
Eventdetail page bug fix

### PR DESCRIPTION
**Pull request by YoonYeoHwan**

### CODE REVIEW

File: web/public/index.html [hunk 1]
```diff
@@ -9,12 +9,12 @@
       name="description"
       content="Web site created using create-react-app"
     />
-    <link rel="apple-touch-icon" href="%PUBLIC_URL%/logo192.png" />
+<!--    <link rel="apple-touch-icon" href="%PUBLIC_URL%/logo192.png" />-->
```
Comment (shberry) [inline web/public/index.html]: Remove if this is not needed

File: web/public/index.html [hunk 2]
```diff
@@ -9,12 +9,12 @@
...
     <!--
       manifest.json provides metadata used when your web app is installed on a
       user's mobile device or desktop. See https://developers.google.com/web/fundamentals/web-app-manifest/
     -->
-    <link rel="manifest" href="%PUBLIC_URL%/manifest.json" />
+<!--    <link rel="manifest" href="%PUBLIC_URL%/manifest.json" />-->
```
Comment (shberry) [inline web/public/index.html]: Same here

File: web/src/components/dateTime.js [hunk 3]
```diff
@@ -1,4 +1,3 @@
-import {Typography} from '@material-ui/core';
 import React, {Component} from 'react';
```
Comment (shberry) [inline web/src/components/dateTime.js]: File name should only include lower case.

"File names must be all lowercase and may include underscores (_) or dashes (-), but no additional punctuation. Follow the convention that your project uses. Filenames’ extension must be .js."

File: web/src/components/dateTime.js [hunk 4]
```diff
@@ -10,25 +9,40 @@ class Datetime extends Component {
...
+        default: dateForm = 'th';
+      }
+    }
+
     return (
-      <Typography variant='body2' style={{marginBottom: 5}}>
-        {month} {day}, {year}, {time}
-      </Typography>
+      <>
+        {/* Feb 5th, 01:30 PM */}
```
Comment (shberry) [inline web/src/components/dateTime.js]: This example doesn't have end time. Maybe add another one?

File: web/src/components/dateTime.js [hunk 5]
```diff
@@ -10,25 +9,40 @@ class Datetime extends Component {
...
+        {month} {day}{dateForm}, {startTime} - {endTime}
```
Comment (shberry) [inline web/src/components/dateTime.js]: Do you know if this is the standard? I've seen '~' and 'to' as well. Can you check which one we should use?

File: web/src/components/eventdetail/bookclub.js [hunk 6]
```diff
@@ -50,23 +47,15 @@ class Bookclub extends Component {
...
-          <Grid item direction='column'>
-            <Typography variant='h6' style={{color: '#009688'}}>
-              Location
-            </Typography>
-            {/* TODO(YoonYeoHwan): Have to fix UI. */}
-            <Typography variant='body2'>
-              {this.props.children.schedule[0].address}
-            </Typography>
-          </Grid>
+
```
Comment (shberry) [inline web/src/components/eventdetail/bookclub.js]: remove

File: web/src/components/eventdetail/eventthumbnail.js [hunk 7]
```diff
@@ -9,7 +9,8 @@ class Eventthumbnail extends Component {
     return (
       <Grid style={{padding: 15, height: 400}}>
         <div style={{height: '100%', width: '100%', overflow: 'hidden'}}>
-          <img style={{height: '100%', width: '100%', objectFit: 'cover'}}
+          <img alt='eventThumbnail'
+            style={{height: '100%', width: '100%', objectFit: 'cover'}}
```
Comment (shberry) [inline web/src/components/eventdetail/eventthumbnail.js]: Some of these can be moved to above line

File: web/public/index.html [hunk 8]
```diff
@@ -24,7 +22,7 @@
       work correctly both with client-side routing and a non-root public URL.
       Learn how to configure a non-root public URL by running `npm run build`.
     -->
-    <title>React App</title>
+    <title>Tiny Social</title>
```
Comment (ethkim) [inline web/public/index.html]: Nice catch. :-)

**Pull request merged**